@xen-orchestra/lite/src/views/pool/PoolDashboardView.vue

```diff
@@ -1,28 +1,27 @@
 <template>
   <div class="pool-dashboard-view">
-    <div class="item">
+    <div class="row first-row">
       <PoolDashboardStatus />
+      <PoolDashboardComingSoon title="Alarms" class="alarms" />
+      <PoolDashboardComingSoon title="Patches" />
     </div>
-    <div class="item">
-      <PoolDashboardStorageUsage />
+    <div class="row">
+      <div class="column">
```

Comment:
Do you think it would be interesting if `<div class="row">` and `<div class="column"` were their own component? for example `<Row>` and `<Column>`?

`class="row"` and `class="column"` will probably be used elsewhere. (host dashboard)

Reply:
Not a bad idea 🤔

Since it's recommended to use multi-words name for components and that these will probably be only used to build various dashboards, I propose `DashboardRow` and `DashboardColumn`.

Reply:
After experimentation, I think that the notion of "row" and "column" is misleading in this case.

It suggests that we need to put columns in rows, which is not the case here.

So I made only 1 component, named `UiCardGroup` which can take an optional `vertical` prop.

Reply:
I agree